packages/contracts/evm-contracts/contracts/token/IInverseProjected1155.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {IERC1155MetadataURI} from "@openzeppelin/contracts/token/ERC1155/extensions/IERC1155MetadataURI.sol";
+import {IERC4906Agnostic} from "./IERC4906Agnostic.sol";
+
+/// @dev A standard ERC1155 that can be burned and has a special uri function accepting a custom base URI.
+interface IInverseProjected1155 is IERC1155MetadataURI, IERC4906Agnostic {
```

Comment:
Maybe we should add tests for this like we did for the inverse projected ERC721?

Reply:
Definitely, will do